packages/evolution-generator/src/common/defaultInputBase.tsx

```diff
@@ -10,36 +10,53 @@ import config from 'chaire-lib-common/lib/config/shared/project.config';
 import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
 import surveyHelper from 'evolution-legacy/lib/helpers/survey/survey';
 import * as surveyHelperNew from 'evolution-common/lib/utils/helpers';
-import * as inputTypes from '../types/inputTypes';
+import {
```

Comment:
So If I understand correctly, these are the same Types that are use directly in the Input Frontend code, so there won't have disparity between the Generator and the Frontend ?

Reply:
> So If I understand correctly, these are the same Types that are use directly in the Input Frontend code, so there won't have disparity between the Generator and the Frontend ?

yes indeed, the objective of this PR is to use only one set of types and make sure it matches the WidgetConfig's type, which are those actually used by the widgets. They are not all correctly or perfectly typed, but at least, we have only one set of types to update.

Anyway, the goal is to move the defaultInputBase data to evolution-common, but the next button action depends on an action that is still in evolution-legacy so we can't do that yet.

Reply:
Oh yeah, I had a few problems with this action button before, mostly compilation problems if I remember correctly.